_translator-files/po/de/Installation-for-iOS.po

```diff
@@ -3,149 +3,150 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2022-05-20 19:13+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.0.1\n"
 
 #. type: YAML Front Matter: lang
 #: ../wiki/en/Installation-for-iOS.md:1
 #, no-wrap
 msgid "en"
-msgstr ""
+msgstr "De"
 
 #. type: YAML Front Matter: layout
 #: ../wiki/en/Installation-for-iOS.md:1
 #, no-wrap
 msgid "wiki"
-msgstr ""
+msgstr "wiki"
 
 #. type: YAML Front Matter: permalink
 #: ../wiki/en/Installation-for-iOS.md:1
 #, no-wrap
 msgid "/wiki/Installation-for-iOS"
-msgstr ""
+msgstr "/wiki/Installation-for-iOS"
 
 #. type: YAML Front Matter: title
 #: ../wiki/en/Installation-for-iOS.md:1
 #, no-wrap
 msgid "Installation for iOS"
-msgstr ""
+msgstr "Installation für iOS"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:9
 msgid "{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"Jamulus verwenden\" branch1=\"Setup - Erste Schritte mit Jamulus\" branch1-url=\"Getting-Started\" %}"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:12
 msgid "# Installation for iOS"
-msgstr ""
+msgstr "# Installation für iOS"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:14
 msgid "Make sure you've already read the [Getting Started](Getting-Started) page."
-msgstr ""
+msgstr "Stelle sicher, dass du die [Erste Schritte](Getting-Started) Seite bereits gelesen hast."
```

Comment:
Probably the English version should reword the "skills". (It shouldn't insult people)
@gilgongo

Reply:
Done.

Reply:
Thanks.